.gitignore

```diff
@@ -159,3 +159,4 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
+.vscode/
```

Comment:
**issue:** No newline at the end of the file.

Reply:
Fixed!